test/Altinn.Profile.Tests/Profile/Validators/CustomRegexForNotificationAddressesTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Altinn.Profile.Models;
+using Altinn.Profile.Validators;
+using Microsoft.VisualBasic;
+using Xunit;
+
+namespace Altinn.Profile.Tests.Profile.Validators
+{
+    public class CustomRegexForNotificationAddressesTests
+    {
+        [Theory]
+        [InlineData("98765432")]
+        [InlineData("9876543200")]
+        [InlineData("")]
+        [InlineData(null)]
+        public void CustomRegex_WhenPhone_IsValid(string input)
```

Comment:
Great use of Theory for this testing. You could do a tad elaboration on the method names, but otherwise looks very good
```suggestion
        public void CustomRegex_WhenPhoneHasAllowedValues_ReturnsValidResult(string input)
```